### PR TITLE
Display checkboxes in ie8

### PR DESCRIPTION
**Pull request by Henni**
#18735 introduced new checkboxes.

This fixes them for ie8.

see issue #18619 

cc @MorrisJobke @davitol @jancborchardt

### CODE REVIEW

File: core/css/styles.css [hunk 1]
```diff
@@ -198,6 +198,11 @@ input[type="checkbox"]:hover+label:before, input[type="checkbox"]:focus+label:be
 	color:#111 !important;
 }
 
+.lte8 input[type="checkbox"] + label:before { background-image: url('../img/actions/checkbox.png'); }
+.lte8 input[type="checkbox"].white + label:before { background-image: url('../img/actions/checkbox-white.png'); }
+.lte8 input[type="checkbox"]:checked + label:before { background-image: url('../img/actions/checkbox-checked.svg'); }
```
Comment (MorrisJobke) [inline core/css/styles.css]: IE8 doesn't support SVG

Comment (Henni) [inline core/css/styles.css]: thanks
missed the last svg

Comment (MorrisJobke) [inline core/css/styles.css]: THe line below too ;)

**Comment by MorrisJobke** [comment]
I just forgot that you should add this to `core/css/fixes.css` Thanks :)

**Comment by MorrisJobke** [comment]
Beside that it shows now the checkboxes in ie8 \o/ :+1:

**Comment by Henni** [comment]
Moved the changes to `fixes.css` and rebased the commit.

**Comment by MorrisJobke** [comment]
Still works

**Comment by jancborchardt** [comment]
:+1: nice!

**Comment by Henni** [comment]
@MorrisJobke do the checkboxes actually work? I just found out that IE8 doesn't support the `:checked` state. http://caniuse.com/#feat=css-sel3

**Comment by MorrisJobke** [comment]
> @MorrisJobke do the checkboxes actually work? I just found out that IE8 doesn't support the :checked state. http://caniuse.com/#feat=css-sel3

:see_no_evil: Never thought about this :( Nope, they don't work.

**Comment by davitol** [comment]
As you bith have said, the checkboxes appear but they do not work. :crying_cat_face:

**Pull request merged**